### ISSUE TITLE
A solution for tailings. 

### ISSUE THREAD

**Issue by AntimatterMemes** [StrangeLoopGames/EcoSuggestions#656]
I propose another solution for storing tailings:
Tailings ponds/impoundments.
The idea is you take tailings, put them in a pond, wait some time for the tailings to settle, then add bacteria (such as methanogens) that will eventually render tailings into (maybe?) sand. 
All of this will take time, of course, just throwing ideas out there, because digging deep underground to store tailings shouldnt be the only way to store them.

**Comment by DragonSupremacy** [comment]
Agreed. Without mods that do add recycling to it, a smith shouldn't have to continuously dig at bedrock for the rest of the map just to store all the tailings from the server's iron/steel needs. There should be at least some way to deal with them properly.

**Comment by BitsBytesGaming** [comment]
I've had this issue before where I was mining my own stone to build my factory floor, making pitch, and trying to advance tech-wise and build the steam truck, but also farming huge amounts of crops for the server. Sometimes there are just too many jobs to do.
